src/marlin_stubs/pause/M70X.cpp

```diff
@@ -238,22 +239,26 @@ static PreheatStatus::Result M1400_no_parser(uint32_t val) {
     case PreheatMode::None:
         return Filaments::CurrentIndex() == filament_t::NONE ? PreheatStatus::Result::DoneNoFilament : PreheatStatus::Result::DoneHasFilament;
     case PreheatMode::Purge:
+        Pause::Instance().StopReset();
```

Comment:
note:
I think table (mask) containing which state should call `Pause::Instance().StopReset();` would be better

Reply:
or just do use one more switch before this one
```
switch (data.Mode()) {
    case PreheatMode:: some_state :
    ...
    case PreheatMode:: some_state :
      Pause::Instance().StopReset();
    default:
      break;
}
```